07.ls_object/lib/screen.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require 'io/console/size'
+
+class Screen
+  def initialize(bulk_created_stats)
+    @file_stats = bulk_created_stats
```

Comment:
```suggestion
    @file_stats = file_stats
```

でいいような。

> ただし、必ず一致させないといけないわけではないので、機械的に対応しないようにしてください。（メソッド定義側の引数名は多少抽象度が上がっても大丈夫なことが多い）
https://bootcamp.fjord.jp/pages/readable-variable-name

Reply:
なるほど、改めて読むと抽象度が多少あがっても読み手は想像がつきますね。
変更してみました。